library/langdet/Text/LanguageDetect.php

```diff
@@ -396,7 +396,7 @@ public function languageExists($lang)
      * Returns the list of detectable languages
      *
      * @access public
-     * @return array        the names of the languages known to this object<<<<<<<
+     * @return array        the names of the languages known to this object
```

Comment:
These extra <<<< confused my manual search (with Midnight Commander) for merge conflicts.